src/DistTime.java

```diff
@@ -2,27 +2,26 @@
 
 public class DistTime {
 
-	private double channelRate;
 	private double packetTime;
 	private int numberOfPackets;
 	private double ipRate;
 
-	public DistTime(int flitWidth, int flitClockCycles, double frequency, int packetSize, int numOfPackets, double rate) {
-		channelRate = frequency * flitWidth;
+	public DistTime(int flitClockCycles, int packetSize, int numOfPackets, double rate) {
 		packetTime = packetSize * flitClockCycles;
 		numberOfPackets = numOfPackets;
-		ipRate = (rate / 100) * channelRate;
+		ipRate = (rate / 100);
 	}
 
-	public ArrayList<String> uniform() {
-		double idleTime = ((channelRate / ipRate - 1) * packetTime);
+	public List<Integer> uniform() {
+		double idleTime = ((1 / ipRate - 1) * packetTime);
```

Comment:
I think you can put the calc of idleTime and totalTime both on constructor and make totalTime the only attribute of this class.